docs/overrides/main.html

```diff
@@ -1,5 +1,10 @@
 {% extends "base.html" %}
 
+{% block announce %}
+  ⚠️ Powertools for Python v2 is coming soon!
```

Comment:
```suggestion
  👋 Powertools for Python v2 is coming soon!
```

Reply:
@rubenfonseca that's the only thing I'd change. Wording is great!

👋 is a more friendly "attention grab" than ⚠️ that could be misunderstood as a problem